script.100fm/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category>
+        <setting label="10000" type="number" id="channel" default="0" visible="false"/>
```

Comment:
I'll try to see what can I do in order to get rid of this file...

@phil65 I would really appreciate it if you could write down the complete list of things that need to be changed in order to have my PR approved. I'm trying to do something good for the community, on my private time, by creating add-ons which enrich Kodi content, but then I find myself spending time in re-submitting PRs for more than a week (and that is after you already found it valid, if you remember, but asked me just to move it to this repo).
I appreciate what you're doing here, I learn a lot from your comments, but I find myself demotivated to work on additional add-ons because of this long process.
Thanks in advance.